crawl-ref/source/mon-place.cc

```diff
@@ -2189,6 +2190,8 @@ static const map<band_type, vector<member_possibilites>> band_membership = {
     { BAND_POLYPHEMUS,          {{{MONS_CATOBLEPAS, 1}},
 
                                  {{MONS_DEATH_YAK, 1}}}},
+    { BAND_CIRI,                {{{MONS_GERALT, 1}},
+                                 {{MONS_YENNEFER, 1}}}},
```

Comment:
I think this has a 50% chance of spawning two Geralts or two Yennefers, but I haven't tested.